weasyl/api.py

```diff
@@ -1,5 +1,7 @@
 # api.py
```

Comment:
If we are removing the ``# <file>`` comments, the following files were missed (looking through just the changed file list here in this PR):

api, avatar, banner, content, define, emailer, embed, errorcode, files, folder, ignoreuser, login, note, premiumpurchase, resetpassword, search, searchtag, shout, welcome